misc/identity-keys/src/identity-keys.ts

```diff
@@ -66,15 +66,16 @@ export class IdentityKeys implements IIdentityKeys {
       try {
         const [pubKeyHex, privKeyHex] = await this.generateIdentityKey();
         const didKey = encodeEd25519Key(pubKeyHex);
+        const domain = new URL(this.keyserverUrl).host;
 
         const cacao: Cacao = {
           h: {
             t: "eip4361",
           },
           p: {
             aud: this.keyserverUrl,
-            statement: "Test",
-            domain: this.keyserverUrl,
+            statement: `${domain} wants you to sign in with your account: ${accountId}`,
```

Comment:
Not really the right statement. This is redundant and is already added [here](https://github.com/WalletConnect/walletconnect-utils/blob/a2f9327d79e02cd5f7c6eed9f5ecb51679f417a5/misc/cacao/src/utils.ts#L37).

Maybe should be `""` to match identity keys spec?

```suggestion
            statement: "",
```